src/pages/products/productDetail/components/review/WriteReviewArea.js

```diff
@@ -0,0 +1,27 @@
+import './WriteReviewArea.scss';
+import ReviewModal from './ReviewModal';
+import { useState } from 'react';
+
+const WriteReviewArea = () => {
+  let [modal, setModal] = useState(false);
+
+  const showModal = () => {
+    setModal(true);
+  };
+  return (
+    <div className="writeReviewArea">
+      {modal ? <ReviewModal setModal={setModal} /> : null}
```

Comment:
```suggestion
      {modal && <ReviewModal setModal={setModal} /> }
```
&& 연산자를 이용한 방법으로도 구현이 가능합니다